detectors/resources/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+	id 'java'
+	id 'com.github.sherter.google-java-format' version '0.9'
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	compile(libraries.opentelemetry_api)
```

Comment:
Gradle deprecated "compile" configuration for "implementation".  Please use that.